packages/reporter/src/lib/events.ts

```diff
@@ -58,6 +58,8 @@ const events: Events = {
     }))
 
     runner.on('reporter:log:add', action('log:add', (log: LogProps) => {
+      if (log.hidden) return
```

Comment:
Is this still needed?

Reply:
technically no, but I left it is a safe-guard for replay